proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -221,6 +221,13 @@ message VscSendTimestamp {
       [ (gogoproto.stdtime) = true, (gogoproto.nullable) = false ];
 }
 
+// ExportedVscSendTimestamps is VscSendTimestamp with chainID info for exporting to genesis
+message ExportedVscSendTimestamp {
```

Comment:
What's the reasoning behind introducing this new message instead of adding a chain_id field to the already existing VscSendTimestamp?

Reply:
See this thread:
https://github.com/cosmos/interchain-security/pull/1076#discussion_r1239689182